noir-projects/noir-protocol-circuits/crates/types/src/merkle_tree/variable_merkle_tree.nr

```diff
@@ -50,17 +51,15 @@ fn get_prev_power_2(value: u32) -> u32 {
 
     let next_power_2 = 2 << next_power_exponent;
     let prev_power_2 = next_power_2 / 2;
-    assert((value == 0) | (value == 1) | (value > prev_power_2));
+    assert(prev_power_2 < value);
```

Comment:
If value equals 0 or 1, the previous power of 2 is completely unconstrained and we could craft a response from `get_next_power_exponent` such that prev_power_2 is anything in [1, 2, 4, 8, 16, ...].